core/test/scripts/VoteGasEstimation.js

```diff
@@ -0,0 +1,121 @@
+const { RegistryRolesEnum, VotePhasesEnum } = require("../../utils/Enums.js");
```

Comment:
I think this script should go in `core/scripts/local` - this is where we're putting scripts that are useful for local development, but not meant for production use.